perseo-main/src/main/resources/perseo-core.properties

```diff
@@ -25,4 +25,5 @@ action.url = http://127.0.0.1:9090/actions/do
 
 # Time in milliseconds (long) to "expire" a "dangling" rule
 rule.max_age= 60000
-
+properties.path = /etc/perseo-core.properties
+action.path = /actions/do
```

Comment:
Both parameters should have an explanatory comment above each one (as rule.max_age does). In addition, I understand the new parameters should be included in documentation/config.md.

In addition, it seems that action.url inclues a part that is the value off action.path. This is a bit weird, isn't it?

Reply:
The change is due to a rule that detects the hardcoded urls and for that reason it was decided to take them out to a configuration file. If the path is considered not to be a url, the change is reversed.

Reply:
It is an URL. However, looking how it is used in Configuration.java

https://github.com/telefonicaid/perseo-core/blob/cff907de5d89aa432543fea0e4dc978370a4f05b/perseo-main/src/main/java/com/telefonica/iot/perseo/Configuration.java#L96-L99

actionPath is just a default to complete the URL in the case it the action URL that comes from  action.url (in the configuration file) or PERSEO_FE_URL (as env var). Thus, yes, probably better to revert this particular change.

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823